tests/unit/utils/toHoursAndMinutes.spec.js

```diff
@@ -0,0 +1,19 @@
+import toHoursAndMinutes from "@/utils/toHoursAndMinutes";
+
+const input = 71;
+const output = toHoursAndMinutes(input);
+
+describe("toHoursAndMinutes", () => {
```

Comment:
1. let's see what the outline of this test looks like:
```
toHoursAndMinutes
  - returns value
  - returns string
  - return string in correct format
```

does it really tell you something about this function? what are the edge cases here?

some suggestion for another outline
```
toHoursAndMinutes
  - returns __(?)__ when number is less than or equal to 0
  - returns time in minutes when number is less than 60
  - return time in hours and minutes when number is greater than or equal to 60
```

2. lets see how particular test looks like:
```js
it("return string in correct format", () => {
    const re = /\dh\s\d{2}min/;
    expect( output ).toMatch( re );
})
```
and compare it to this:
```js
it("return string in correct format", () => {
    expect( toHoursAndMinutes(71) ).toMatch( "1h 11min" );
})
```
which test tells you something about how this function actually works?

Reply:
thank you for this!!! I really appreciate the essay!!! 💚  I'll make a point of making the tests more descriptive from functioning point of view and testing actual functionality and edge cases :)